src/components/addpeer/WindowsTab.tsx

```diff
@@ -3,16 +3,18 @@ import React, {useState} from 'react';
 import { Button } from "antd";
 import TabSteps from "./TabSteps";
 import { StepCommand } from "./types"
+import {getConfig} from "../../config";
```

Comment:
Codacy has a fix for the issue: [Replace `getConfig` with `·getConfig·`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9561089)
```suggestion
import { getConfig } from "../../config";
```